deployment-manager/src/main/java/com/example/deploymentmanager/DeploymentManagerDemo.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.deploymentmanager;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.deploymentmanager.DeploymentManager;
+import com.google.api.services.deploymentmanager.model.*;
+
+import java.io.*;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class DeploymentManagerDemo {
+
+  private static String
+          GOOGLE_PROJECT_ID,
+          DEPLOYMENT_NAME,
+          CONFIG_YAML = "yaml/deployment-manager-config.yaml";
+  
+  private static DeploymentManager _dm = null;
+
+  public static void main(String args[]) {
+    init();
+    runDemo();
+    // cleanup(); // Optionally delete resources.
+  }
+
+  /*
+   Initialize Google api.
+  */
+  public static void init() {
```

Comment:
This entire function should probably just be refactored into a "createClient" that returns the client. The other function can just call this function to create the client and use it from there.

Reply:
Example of this being done here: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/81243cb46e3ea02a177dfb3fc04d60def6202138/healthcare/v1beta1/src/main/java/snippets/healthcare/datasets/DatasetCreate.java#L80-101

Reply:
Much cleaner. Done